firmware/brake/kia_soul_ps/brake_control_module.ino

```diff
@@ -142,66 +142,115 @@ int16_t solenoid_left_actuation_duty_min;
 int16_t solenoid_left_release_duty_max;
 int16_t solenoid_left_release_duty_min;
 
+// *****************************************************
+// Function:    calculate_solenoid_left_actuation_duty_cycle
+// 
+// Purpose:     Calculate pressure value
+// 
+// Returns:     int16_t - the scaled pressure value
+// 
+// Parameters:  float pre - the pressure value
+// 
+// *****************************************************
 int16_t calculate_solenoid_left_actuation_duty_cycle( float pre )
 {
   int16_t scaled = abs( pre ) * 512;
   int16_t scale =  map( scaled, 0, 1024, solenoid_left_actuation_duty_min, solenoid_left_actuation_duty_max );
   return scale;
 }
 
-
+// *****************************************************
+// Function:    calculate_solenoid_left_release_duty_cycle
+// 
+// Purpose:     Calculate pressure value
+// 
+// Returns:     int16_t - the scaled pressure value
+// 
+// Parameters:  float pre - the pressure value
+// 
+// *****************************************************
 int16_t calculate_solenoid_left_release_duty_cycle( float pre )
 {
   int16_t scaled = abs( pre ) * 512;
   int16_t scale =  map( scaled, 0, 1024, solenoid_left_release_duty_min, solenoid_left_release_duty_max );
   return scale;
 }
 
+// *****************************************************
+// Function:    pressure_to_voltage
+// 
+// Purpose:     Convert pressure to voltage
+// 
+// Returns:     float - voltage
+// 
+// Parameters:  int16_t - pressure in MPa
+// 
+// *****************************************************
 float pressure_to_voltage( int16_t MPa )
 {
-	// convert MPa pressure to equivalent voltage
     return ( MPa + 217.1319446 ) / 505.5662053;
 }
 
+// *****************************************************
+// Function:    voltage_to_pressure
+// 
+// Purpose:     Convert voltage to pressure
+// 
+// Returns:     int16_t - pressure in MPa
+// 
+// Parameters:  float - volts
+// 
+// *****************************************************
 int16_t voltage_to_pressure( float voltage )
 {
-    // convert voltage reading from sensors to pressure in MPa
     return ( voltage * 505.5662053 ) - 217.1319446;
 }
 
-// convert the ADC reading (which goes from 0 - 1023) to a voltage (0 - 5V):
+// *****************************************************
+// Function:    convert_to_voltage
+// 
+// Purpose:     Convert the ADC reading
+//				(which goes from 0-1023)
+//				to a voltage (0 - 5V)
+// 
+// Returns:     float - volts
+// 
+// Parameters:  int16_t - ADC reading 
+// 
+// *****************************************************
 float convert_to_voltage( int16_t input )
```

Comment:
This is a badly named function.  Can we look up with this is doing and name it better?  Maybe "scale_analog_input" or something like that would work better?